subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/inputs/undeclared/UndeclaredBuildInputsIntegrationTest.groovy

```diff
@@ -592,6 +600,58 @@ class UndeclaredBuildInputsIntegrationTest extends AbstractConfigurationCacheInt
         outputContains("value = value")
     }
 
+    @Issue("https://github.com/gradle/gradle/issues/25044")
```

Comment:
:tada: thanks, I forgot to mention this, but issue references are very nice to have.